src/index.css

```diff
@@ -8,3 +8,29 @@ body,
 #root > div {
   height: 100%;
 }
+
+/* Global css-variables - can and should be used when coloring/shadowing anything */
+:root {
+  --primary-dark: #007ac1;
+  --primary-normal: #03a9f4;
+  --primary-light: #67daff;
+
+  --secondary-dark: #b0003a;
+  --secondary-normal: #e91e63;
+  --secondary-light: #ff6090;
+
+  --positive-normal: #219166;
+  --positive-dark: #087f23;
+  --positive-light: #14d596;
+
+  --negative-normal: #f62d1d;
+  --negative-dark: #8e0000;
+  --negative-light: #ff5f52;
+
+  --shadow-primary: 0 0 2px 1px black;
+  --shadow-secondary: 0 0 1.5px 0.5px black;
+  --shadow-big: 0 0 4px 1px black;
+
+  --big-radius: 5px;
+  --small-radius: 2px;
+}
```

Comment:
I think the colors might be a bit dark? I think it changes a lot if the text is white, actually. See image for proposal. The text for `North` `East` `Surge` `Sway` etc could be black tho.
In general; if we have dark colors we should have bright text, if we have bright colors we should have dark text.
![image](https://user-images.githubusercontent.com/37441744/67159202-fa57e980-f341-11e9-892c-78158082ccbc.png)

Reply:
Black on the dark blue passes the contrast checker, but I agree that white probably looks better. I will change it and then we can merge and iterate later 👍 

![image](https://user-images.githubusercontent.com/31652936/67159303-f24c7980-f342-11e9-9ff1-44590da377e4.png)

Reply:
![image](https://user-images.githubusercontent.com/31652936/67159334-4ce5d580-f343-11e9-8296-c30875fe785b.png)

![image](https://user-images.githubusercontent.com/31652936/67159337-5707d400-f343-11e9-98f3-57124b41db1a.png)

Reply:
I think maybe black looks better 🤔